### PR TITLE
use bitwise operator

### PR DESCRIPTION
**Pull request by ZhouHansen**
Just use the existed use bitwise operator.

## Checklist

- [x] tests pass
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added

## Context
None

## Semver Changes
None

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -295,19 +295,4 @@ fn is_even(n: usize) -> bool {
...
-
-#[test]
-fn shifts_bits_right() {
-  assert_eq!(shift_right(0), 0);
-  assert_eq!(shift_right(9), 4);
-  assert_eq!(shift_right(12), 6);
-  assert_eq!(shift_right(13), 6);
-  assert_eq!(shift_right(14), 7);
-}
+}
```
Comment (yoshuawuyts) [inline src/lib.rs]: Could you restore this newline?

Comment (ZhouHansen) [reply src/lib.rs]: Is there a problem?

Comment (ZhouHansen) [reply src/lib.rs]: OK, I've done it.

**Comment by ZhouHansen** [comment]
Done it, but again omit signoff... so another PR created.

**Comment by yoshuawuyts** [comment]
Closed by #5

**Pull request closed without merging**